src/main/java/ui/listpanel/ListPanelCell.java

```diff
@@ -54,6 +52,20 @@ public void updateItem(TurboIssue issue, boolean empty) {
         this.setId(issue.getRepoId() + "_col" + parentPanelIndex + "_" + issue.getId());
     }
 
+    private void updateStyleToMatchStatus(TurboIssue issue){
+        if (!issue.isOpen()){
+            // adds closed issue styling from hubturbo.css if the style is not contained in the ListCell
+            if (!getStyleClass().contains("issue-cell-closed")) {
+                getStyleClass().add("issue-cell-closed");
+            }
+        } else {
+            // For open issues, closed issue styling is removed if it is contained in the cell.
+            if (getStyleClass().contains("issue-cell-closed")) {
+                getStyleClass().remove("issue-cell-closed");
+            }
+        }
+    }
```

Comment:
The style class string `"issue-cell-closed"` and `getStyleClass()` is repeated 4 times. May be declare as local variables? Something like this?
```
currentStyleClass = ...
String closedStyle = "issue-cell-closed";
boolean isCurrentStyleClosed = getStyleClass().contains(closedStyle);

if(issue.isOpen() && isCurrentStyleClosed ) {
    currentStyleClass.remove(closedStyle);
```

Now that the code is clearer, may be no need for the comments?
Get @lejolly to have a look after revising.